rerun_py/rerun_sdk/rerun2/archetypes/points2d.py

```diff
@@ -0,0 +1,114 @@
+# NOTE: This file was autogenerated by re_types_builder; DO NOT EDIT.
+
+from __future__ import annotations
+
+__all__ = ["Points2D"]
+
+
+from dataclasses import dataclass
+
+from rerun2 import components
+
+
+@dataclass
+class Points2D:
+    """A 2D point cloud with positions and optional colors, radii, labels, etc."""
+
+    points: components.Point2DArray
+    """
+    All the actual 2D points that make up the point cloud.
+    """
+
+    radii: components.RadiusArray | None = None
+    """
+    Optional radii for the points, effectively turning them into circles.
+    """
+
+    colors: components.ColorArray | None = None
+    """
+    Optional colors for the points.
+
+    The colors are interpreted as RGB or RGBA in sRGB gamma-space,
+    As either 0-1 floats or 0-255 integers, with separate alpha.
+    """
+
+    labels: components.LabelArray | None = None
+    """
+    Optional text labels for the points.
+    """
+
+    draw_order: components.DrawOrderArray | None = None
+    """
+    An optional floating point value that specifies the 2D drawing order.
+    Objects with higher values are drawn on top of those with lower values.
```

Comment:
I thought the linter gets angry when the synopsis is not a single line?
Might be interesting to explore having warnings from the code generator for "bad style" 🤔 (future work)